### PR TITLE
Drop all references to system-setup

### PR DESCRIPTION
**Pull request by ogayot**

### CODE REVIEW

**Review by dbungert — COMMENTED** [review]
Posting this now for the one comment, I still need to read runtests.sh

File: subiquity/server/controllers/cmdlist.py [hunk 1]
```diff
@@ -108,20 +108,14 @@ def __init__(self, app):
         super().__init__(app)
         self.syslog_id = app.log_syslog_id
 
-        try:
-            hooks_dir = self.app.opts.postinst_hooks_dir
-        except AttributeError:
```
[the remark this answers is not in the document]
Comment (ogayot) [reply subiquity/server/controllers/cmdlist.py:113]: I added this `try ... except` construct specifically for `system_setup` (see the comment in the except block) so IMO we should drop it. But I can make it a separate commit if that helps?

[the remark this answers is not in the document]
Comment (dbungert) [reply subiquity/server/controllers/cmdlist.py:113]: I like that.   I do want to drop the try/except but if it became a problem, bisect would find it.

**Review by dbungert — APPROVED** [review]
LGTM

**Review by mwhudson — APPROVED** [review]
What a nice diffstat for PR 2000!

**Review by CarlosNihelton — APPROVED** [review]
Couldn't find a single missed reference, so LGTM! :tada:
Thanks for addressing this!

As a side note, you may want to revert #1305, as that was introduced to allow system_setup to talk over TCP, unless you foresee other reasons for allowing variants to override their `start_site` behaviour.

**Comment by ogayot** [comment]
> Couldn't find a single missed reference, so LGTM! 🎉 Thanks for addressing this!
> 
> As a side note, you may want to revert #1305, as that was introduced to allow system_setup to talk over TCP, unless you foresee other reasons for allowing variants to override their `start_site` behaviour.

Good point, I've reverted this change as well, thanks!

**Pull request merged**